node/src/isolate/external.cpp

```diff
@@ -434,4 +485,25 @@ external_t::spawn(const std::string& path,
     return std::unique_ptr<api::cancellation_t>(new api::cancellation_wrapper(load));
 }
 
+
+// TODO: is concurrency supported?
+// it seems that spool and spawn concurrent access is ensured by state machine.
+void
+external_t::metrics(const dynamic_t& query, std::shared_ptr<api::metrics_handle_base_t> handle) const {
+
+    auto load = std::make_shared<metrics_load_t>(inner, query, handle);
```

Comment:
For code-base consistency with spool/spawn, initial release has metrics load queue, but it was removed as redundant for polling task, as it seems not crucial to loose some payload for it.